README.md

```diff
@@ -1,310 +1,320 @@
-
-It is possible to configure the conda environment to set environmental variables on activation. Providing instructions to set PATH to include the CUDA toolkit bin directory and LD_LIBRARY_PATH to include the CUDA lib64 directory will be helpful.
-
-
-
-```bash
-cd  ~/anaconda3/envs/cugraph_dev
-
-mkdir -p ./etc/conda/activate.d
-mkdir -p ./etc/conda/deactivate.d
-touch ./etc/conda/activate.d/env_vars.sh
-touch ./etc/conda/deactivate.d/env_vars.sh
-```
-
-
-
-Next the env_vars.sh file needs to be edited
-
-```bash
-vi ./etc/conda/activate.d/env_vars.sh
-
-#!/bin/bash
-export PATH=/usr/local/cuda-10.0/bin:$PATH # or cuda-9.2 if using CUDA 9.2
-export LD_LIBRARY_PATH=/usr/local/cuda-10.0/lib64:$LD_LIBRARY_PATH # or cuda-9.2 if using CUDA 9.2
-```
-
-
-
-```
-vi ./etc/conda/deactivate.d/env_vars.sh
-
-#!/bin/bash
-unset PATH
-unset LD_LIBRARY_PATH
-```
-
-
-
-
-
-
-
-------
-
-
-
-## <div align="left"><img src="img/rapids_logo.png" width="265px"/></div> Open GPU Data Science
-
-The RAPIDS suite of open source software libraries aim to enable execution of end-to-end data science and analytics pipelines entirely on GPUs. It relies on NVIDIA® CUDA® primitives for low-level compute optimization, but exposing that GPU parallelism and high-bandwidth memory speed through user-friendly Python interfaces.
-
-<p align="center"><img src="img/rapids_arrow.png" width="80%"/></p>
-
-### Apache Arrow on GPU
-
-The GPU version of [Apache Arrow](https://arrow.apache.org/) is a common API that enables efficient interchange of tabular data between processes running on the GPU. End-to-end computation on the GPU avoids unnecessary copying and converting of data off the GPU, reducing compute time and cost for high-performance analytics common in artificial intelligence workloads. As the name implies, cuDF uses the Apache Arrow columnar data format on the GPU. Currently, a subset of the features in Apache Arrow are supported.
+# <div align="left"><img src="img/rapids_logo.png" width="90px"/>&nbsp;cuGraph - GPU Graph Analytics</div>
+
+[![Build Status](http://18.191.94.64/buildStatus/icon?job=cugraph-master)](http://18.191.94.64/job/cugraph-master/)  [![Documentation Status](https://readthedocs.org/projects/cugraph/badge/?version=latest)](https://cugraph.readthedocs.io/en/latest/)
+
+The [RAPIDS](https://rapids.ai) cuGraph library is a collection of graph analytics that process data found in GPU Dataframe - see [cuDF](https://github.com/rapidsai/cudf).  cuGraph aims at provides a NetworkX-like API that will be familiar to data scientists, so they can now build GPU-accelerated workflows more easily.
+
+ For more project details, see [rapids.ai](https://rapids.ai/).
+
+**NOTE:** For the latest stable [README.md](https://github.com/rapidsai/cudf/blob/master/README.md) ensure you are on the `master` branch.
+
+
+
+
+
+
+
+## Getting cuGraph
+### Intro
+There are 4 ways to get cuGraph :
+1. [Quick start with Docker Demo Repo (Coming Soon)](#quick)

```

Comment:
We say (Coming Soon) on 3 of these.  Is the expectation that this will be available for 0.6?  How soon?  It seems like some sort of time frame (less ambiguous than soon) would be helpful here.

Reply:
Those will be there for 0.6 - I believe that we should include them now but I defer to @taureandyernv